### ISSUE TITLE
Error sharing while using email addresses in angled brackets

### ISSUE THREAD

**Issue by TEMEA-Landau** [nextcloud/polls#3036]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on Github _(I've searched it)_.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### What went wrong, what did you observe?

Hallo Poll-Team!
Leider ist es mir nicht möglich in der Version 5.2.0 unter dem Punkt Teilen eine einzelne Freigabe zu erteilen. Bei vorherigen Umfragen konnte ich immer nicht interne Personen über das Teilen zu einer nicht öffentlichen Umfrage einladen. Ich arbeite sehr gerne  mit der Umfrage. Könnt ihr das bitte wieder ändern? Meine Nextcloud-Version ist die Nextcloud Hub 4 26.0.4.
Vielen Dank und viele Grüße

### What did you expect, how polls should behave instead?

Bisher war es immer so, dass ich Name Vorname <E-Mail@adresse> unter dem Register eingeben konnte und nach dem Drücken der Enter-Taste die Daten übernommen wurden. Dies ist nicht mehr der Fall, es passiert nichts!

### What steps does it need to replay this bug?

1. Dass das Teilen der Umfrage über die manuelle Eingabe wieder möglich ist; wie oben beschrieben.
2. Eine Aktualisierung über Nextcloud

### Installation method

Installed/updated from the appstore (Apps section of your site)

### Installation type

First time installation

### Affected polls version

5.2.0

### Which browser did you use, when experiencing the bug?

- [X] Firefox
- [ ] Chrome
- [ ] Chromium/Chromium based (i.e. Edge)
- [ ] Safari
- [ ] Other/Don't know

### Other browser

Edge

### Add your browser log here

```shell
JQMIGRATE: Migrate is installed, version 3.4.0
index.es.js:2337  Proxying an event bus of version 3.0.2 with 1.3.0
e @ index.es.js:2337
2globals.js:59  jQuery is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
In @ globals.js:59
index.es.js:2337  Proxying an event bus of version 3.0.2 with 1.3.0
e @ index.es.js:2337
session-heartbeat.js:103 session heartbeat polling started
init-collections.js:42  OCA.Sharing.ShareSearch not ready
(anonym) @ init-collections.js:42
index.es.js:2337  Proxying an event bus of version 3.0.2 with 1.3.0
e @ index.es.js:2337
4globals.js:59  $ is deprecated: The global jQuery is deprecated. It will be removed in a later versions without another warning. Please ship your own.
In @ globals.js:59
index.js:319 OC.requestToken changed 1WkHjOxfpBGq7EqN90xxnXHZaVdZeO8wq+J8bCUPwL4=:gyA//bVr8Efyphn7ljhByUOOUS4SHK5+/JImFHxusPM=
```

### NC version

Nextcloud 26 (Nextcloud Hub 4)

### PHP engine version

Other/Don't know

### Database engine

Other/Don't know

### Which user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other/Don't know

### Add your nextcloud server log here

```shell
Weiß ich leider nicht, da meine Nextcloud über eine Nextbox läuft!
```

### Additional environment informations

Weiß ich leider nicht, da meine Nextcloud über eine Nextbox läuft!

**Comment by dartcafe** [comment]
Thanks for you participation, but please be aware that the language used here is english. This way also non german users can understand the issues.

Regarding your issue: I cant reproduce this issue, after entering an email address, I can simply add the share. 

enter email address | select email address
-|-
![grafik](https://github.com/nextcloud/polls/assets/26707476/be1ad590-9acd-4127-9d4a-a0648eeeda69)|![grafik](https://github.com/nextcloud/polls/assets/26707476/893c6ed0-38bb-4ab2-b55f-2c79e678b4fd)

Maybe you can add some more information or screenshots, to understand your issue.

**Comment by TEMEA-Landau** [comment]
Thanks for the quick reply!!!
Sorry! I had the page translated automatically and unfortunately I had written it in German.
I found the solution to my problem. The original syntax I used for sharing was:

![image](https://github.com/nextcloud/polls/assets/143071047/3c0be789-cfff-4bf8-922d-5be0cb90ab65)

Thus, the child's name appeared as the entry and below it was the parents' email address as an entry in the list of shared contacts.
In this version, the square brackets can be omitted and it works again. Now I have my usual view again and can do my sports festival planning as usual. Thank you!
![image](https://github.com/nextcloud/polls/assets/143071047/3387bc29-f391-49fe-83fe-36c09a6de52f)

Best regards
Thorsten

**Comment by dartcafe** [comment]
Ah ok. Thanks. Now I understand. But the bracket version should work. I will investigate this later.

**Comment by dartcafe** [comment]
Unfortunately this behavior is caused from the Nextcloud lib component.

**Comment by dartcafe** [comment]
Waing for https://github.com/nextcloud-libraries/nextcloud-vue/pull/4653 to xome in

**Comment by dartcafe** [comment]
This will be solved with nextcloud 28 and be available after Polls version 6.0 is out.
